packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -60,6 +67,32 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
     };
   }, [cameraSwitcherCallback, cameraSwitcherCameras]);
 
+  useMemo(() => {
+    const onPersonJoined = (e) => {
+      setAnnouncerString(locale.participantJoinedNoticeString);
+      setInterval(() => {
+        setAnnouncerString('');
+      }, 5000);
+    };
+    adapter.on('participantsJoined', onPersonJoined);
+    return () => {
+      adapter.off('participantsJoined', onPersonJoined);
+    };
+  }, []);
+
+  useMemo(() => {
```

Comment:
these add a lot of lines to the render function, consider a creating a helper hook that just returns the string
```
const announcerString = useParticipantJoinedAnnouncement();
```